### DISCUSSION TITLE
Add an option to "Add existing file" to a project

### DISCUSSION THREAD

**Discussion by MangelMaxime** [ionide/ionide-vscode-fsharp#1772] [Ideas]
Currently we can only add new file to the project:

![image](https://user-images.githubusercontent.com/4760796/185759033-8322ea9a-235f-4b5d-a061-f9ae473c79be.png)

It would be nice to also have the possibility to add an existing file

![image](https://user-images.githubusercontent.com/4760796/185759052-03fa5a86-ae06-497b-a928-0051806f50f5.png)

I see to way of doing it:

1. Compute on VSCode side the relative path to add and use the existing `fsproj/addFile` command
2. Add a new command to FSAC called `fsproj/addExistingFile` and do all the logic on FSAC side

Would you be ok to add this feature? Which solution would you prefer to use?

**Comment by Krzysztof-Cieslak** [comment]
Sounds good to me, and I'd rather have logic on FSAC side.
